mysql-test/suite/galera/t/galera_migrate.test

```diff
@@ -226,32 +208,18 @@ DROP TABLE t1;
 --connection node_2
 DROP TABLE t1;
 
---connection node_4
+--connection node_3
 SET GLOBAL wsrep_provider = 'none';
-SET GLOBAL wsrep_sst_method = 'rsync';
```

Comment:
PXC 5.6 doesn't support the rsync sst method?

Reply:
fixed